src/components/chats/chat/ChatMessages.vue

```diff
@@ -1,93 +1,119 @@
+            </unnnic-chats-message>
+            <template v-for="media in message.media">
+              <unnnic-chats-message
+                v-if="isMedia(media) && !isGeolocation(media)"
+                :key="media.created_on"
+                :ref="`message-${message.uuid}`"
+                :type="message.user ? 'sent' : 'received'"
+                :class="['chat-messages__message', message.user ? 'sent' : 'received']"
+                :mediaType="isImage(media) ? 'image' : isVideo(media) ? 'video' : 'audio'"
+                :time="new Date(message.created_on)"
+                :status="messageStatus({ message })"
+                @click="resendMedia({ message, media })"
+              >
+                <img
+                  v-if="isImage(media)"
+                  class="media image"
+                  :src="media.url || media.preview"
+                  @click="openFullScreen(media.url)"
+                  @keypress.enter="openFullScreen(media.url)"
+                />
+                <video-player
+                  v-else-if="isVideo(media)"
+                  class="media"
+                  :src="media.url || media.preview"
+                />
+                <unnnic-audio-recorder
+                  v-else-if="isAudio(media)"
+                  ref="audio-recorder"
+                  class="media audio"
+                  :src="media.url || media.preview"
+                  :canDiscard="false"
+                  :reqStatus="messageStatus({ message, media })"
+                  @failed-click="resendMedia({ message, media })"
+                />
+              </unnnic-chats-message>
+              <unnnic-chats-message
+                v-else-if="!isMedia(media)"
+                :key="media.created_on"
+                :ref="`message-${message.uuid}`"
+                :type="message.user ? 'sent' : 'received'"
+                :class="['chat-messages__message', message.user ? 'sent' : 'received']"
+                :time="new Date(message.created_on)"
+                :documentName="media.url?.split('/').at(-1) || media.file.name"
+                :status="messageStatus({ message })"
+                @click="documentClickHandler({ message, media })"
+              />
+            </template>
+          </template>
+        </template>
       </section>
     </section>
-    <div style="position: sticky; bottom: 0px; background-color: white">
-      <section v-if="!room.is_active" class="chat-messages__room__divisor">
-        <div class="chat-messages__room__divisor__line" />
-        <span class="chat-messages__room__divisor__label">{{ $t('chat_closed_by.agent') }}</span>
-        <div class="chat-messages__room__divisor__line" />
-      </section>
-      <section>
-        <div v-if="room.is_waiting" class="chat-messages__room__transfer-info">
-          {{ $t('waiting_answer.waiting_cliente_answer') }}
-        </div>
-      </section>
+    <!-- Closed chat tags  -->
+    <!-- <chat-feedback
```

Comment:
Dead code?

Reply:
No, this code is commented to handle future updates with the backend